### DISCUSSION TITLE
Apply  time-varying force or velocity field to the cell

### DISCUSSION THREAD

**Discussion by WONGJS** [UvaCsl/HemoCell#24] [General discussion]
Hello

Could you please tell me how to apply a time-varying force or velocity field to a cell in the Hemocell?

Thank you

**Comment by gzavo** [comment]
Hi, you can simply impose a new force-field at every time-step.
You set the external force field like this: [pipeflow example](https://github.com/UvaCsl/HemoCell/blob/08cb2ca397d1b33a1ecc2b467f21d5ac3503bfb4/examples/pipeflow/pipeflow.cpp#L112)
You can set it to a different value every iteration.
